fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,567 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "CCT Customer Service Requests",
+    "description": "This section provides an overview of all aspects of the application programmers’ interface, including registration to use the interface, the various resources that constitute the interface and how these should be orchestrated into an application that will deliver consistent and reliable results.",
+    "contact": {
```

Comment:
can we actually just leave the `"contact"` section out?

Reply:
Contac has been taken out.